src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs

```diff
@@ -44,6 +44,8 @@ public Image()
 
 		private void OnImageFailed(object sender, RoutedEventArgs e)
 		{
+			Console.WriteLine($"Image failed {e}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)